src/externs.ts

```diff
@@ -518,6 +518,11 @@ export function generateExterns(
           debugLocationStr(exportDeclaration, namespace)}\n`);
       return;
     }
+    if (ts.isNamespaceExport(exportDeclaration.exportClause)) {
+      emit(`\n// TODO(tsickle): export * as declaration in ${
```

Comment:
can you open an issue to track adding support?

I think it'd be something like the assignment in the loop below.

Reply:
Filed #1135 and added a TODO!